### ISSUE TITLE
Start a game with a friend who has never logged in to the app

### ISSUE THREAD

**Issue by romac** [reminisceme/backend#39]
Currently, we've only been creating games between two users who've already logged in to the app, and who thus have a `auth_user` row present in the database.

What happens when we'll run the app inside of a Facebook canvas, and want to invite a user who's never used the app before?

**Comment by JRappaz** [comment]
Yep, it's a tough one... We could create a dummy user object with just Facebook id and fill the missing fields when the user actually logs in. Will further inspect but do we need it for the demo?

**Comment by romac** [comment]
Yeah that seems like the way to go (we'll have to investigate how/if social_auth can handles that).

Not needed for the demo, just thought of it and thus opened an issue to not forget about it :)
